Automattic/wp-calypso
# Marketplace: Create DiscoveryPage component

#### Proposed Changes
Following pdh6GB-14m-p2 we have to create a component that encapsulates the state and renders the Discovery Page

#### Screenshot
<img width="1239" alt="Screen Shot 2022-06-10 at 5 38 55 PM" src="https://user-images.githubusercontent.com/1035546/173159323-bcb0694d-53ea-40df-8813-ea3a2912a0c4.png">

#### Testing Instructions

1. Go to `/plugins`
2. Check if Discover tab works as it used to work before this PR applied

#### Checklist

- [x] Group state selection and data fetch logic inside this component
- [x] Page should look the same as it was before the refactor
- [x] It should live in it's own folder

#### Pre-merge Checklist






Fixes #64541
